### ISSUE TITLE
Some packages don't list all versions

### ISSUE THREAD

**Issue by angularsen** [praeclarum/FuGetGallery#26]
Some packages list only a few of the versions listed on nuget.org:
https://www.fuget.org/packages/UnitsNet (`3.92.0-3.99.0` vs `1.2.0-3.99.0`) 
https://www.fuget.org/packages/Humanizer (`2.1.0-2.2.0` vs `0.1.0-2.2.0`)

While other packages seems to list all versions:
https://fuget.org/packages/Polly
https://fuget.org/packages/SkiaSharp

I don't immediately see any common pattern on the breaking point for the packages missing versions, neither on version, dates or target platforms. Any ideas?

**Comment by praeclarum** [comment]
Yes, if there are a lot of releases, I only grab the first bunch so some go missing. I will fix this.

**Comment by mjkkirschner** [comment]
@praeclarum is there a workaround- say to perform an API diff between versions one knows exist?

**Comment by loic-sharma** [comment]
By the way, there's an API that lists all (including unlisted) versions of a package: https://docs.microsoft.com/en-us/nuget/api/package-base-address-resource#enumerate-package-versions

For example: https://api.nuget.org/v3-flatcontainer/newtonsoft.json/index.json
